actors/builtin/miner/miner_state.go

```diff
@@ -525,16 +939,25 @@ func quantizeUp(e abi.ChainEpoch, unit abi.ChainEpoch) abi.ChainEpoch {
 	return e - remainder + unit
 }
 
-func sectorKey(e abi.SectorNumber) adt.Keyer {
+// Rounds e to the nearest exact multiple of the quantization unit, rounding down.
+func quantizeDown(e abi.ChainEpoch, unit abi.ChainEpoch) abi.ChainEpoch {
+	remainder := e % unit
```

Comment:
This and `quantizeUp` will panic when `unit == 0` which is a possible (but unlikely) scenario given the defition here: https://github.com/filecoin-project/specs-actors/pull/286/files#diff-2580e4b82238370ee854a52e8d5428e7R37